verification/src/compiler/version.rs

```diff
@@ -209,4 +266,141 @@ mod tests {
             )
         );
     }
+
+    fn to_versions(versions: Vec<&str>) -> Vec<CompilerVersion> {
+        versions
+            .iter()
+            .map(|s| CompilerVersion::from_str(s).expect("invalid version"))
+            .collect()
+    }
+
+    fn test_shuffle_and_sort(sorted: Vec<&str>) {
+        let sorted_versions = to_versions(sorted.clone());
+        let mut shuffled_versions = sorted_versions.clone();
+        shuffled_versions.shuffle(&mut thread_rng());
+        shuffled_versions.sort();
+        let shuffled: Vec<String> = shuffled_versions.iter().map(|v| v.to_string()).collect();
+        assert_eq!(shuffled, sorted);
```

Comment:
Why not to compare just `shuffled_versions` and `sorted_versions`?

Reply:
in case of wrong order, test will show unreadable error:
<img width="1338" alt="изображение" src="https://user-images.githubusercontent.com/41516657/174546455-ece35b32-a616-4611-a447-3160e4c2d8f3.png">

Reply:
Could you please add a small comment describing that?